vc_vidyo/indico_vc_vidyo/static/js/vc_vidyo.js

```diff
@@ -12,12 +12,7 @@ $(function() {
             url: $this.data('href'),
             method: 'POST',
             complete: IndicoUI.Dialogs.Util.progress(),
-            contentType: 'application/json',
-            data: JSON.stringify({
-                data: {
-                    owner: ['User', $('body').data('userId')]
-                }
-            })
+            contentType: 'application/json'
```

Comment:
not needed anymore, you are not sending and payload so no need to specify a content type